examples/graphql-interfaces/graphql_interfaces.bal

```diff
@@ -15,48 +15,28 @@ public type Animal distinct service object {
     resource function get name() returns string;
 };
 
-// Define another interface `Mammal`, that implements `Animal` interface
-public type Mammal distinct service object {
-
-    // This denotes that this interface implements the `Animal` interface
-    *Animal;
-
-    // Add an additional field to the `Mammal` interface
-    resource function get call() returns string;
-};
-
-// Define the `Leopard` class implementing the `Mammal` interface
+// Define the `Leopard` class implementing the `Animal` interface.
 public distinct service class Leopard {
 
-    // This denotes that this object implements the `Mammal` interface
-    *Mammal;
+    // This denotes that this object implements the `Animal` interface.
+    *Animal;
 
-    // Since this object implements the `Mammal` interface and the `Mammal` interface implements the
-    // `Animal` interface, this object must implement the fields from the `Animal` interface
+    // Since this object implements the `Animal` interface, this object must implement the fields of the `Animal` interface.
```

Comment:
The character limit for a line in a BBE is 100 characters, right? @praneesha Did we change that?

IIRC, previously, there was a validation for that. Seems like that's not running anymore.

Reply:
Fixed with [`2f613bb`](https://github.com/ballerina-platform/ballerina-distribution/pull/3431/commits/2f613bb552c4d09ec049d6ee7af29f035d8104ff)